fr/administrateur/fonctionnalites-logistiques/configuration/zones.md

```diff
@@ -18,5 +18,5 @@ Pour créer les zones, vous devez d'abord les enregistrer au format [GeoJSON](ht
 3. Cliquez sur **Save > GeoJSON** pour télécharger le fichier
 
 ---
-
-![Dessiner un polygône](/img/help/zones/draw_polygon.png "Dessiner un polygône")
+<!-- ![Dessiner un polygône](/img/help/zones/draw_polygon.png "Dessiner un polygône") -->
+![Utilisation de geojson.io]({{ "/assets/images/peekGeoJSON.gif" | absolute_url }})
```

Comment:
- Why is there a `+` at the beginning of the line?
- The filename is in camel case, it should be `peek_geojson.gif`